app/jobs/send_purge_warnings_job.rb

```diff
@@ -14,8 +14,8 @@ def perform(*_args)
 
     # Loop through and send each admin information about their purge eligible monitorees
     recipients.each do |user|
-      # Skip for USA admins
-      next if user.jurisdiction&.name == 'USA'
+      # Skip for USA admins and locked users
+      next if user.jurisdiction&.name == 'USA' || !user.locked_at.nil?
```

Comment:
I think this works fine but thoughts on using `user.locked_at.present?` instead to avoid a double negative?